docs/guides/chip_tool_guide.md

```diff
@@ -778,25 +841,74 @@ following command:
 $ ./chip-tool discover commissioners
 ```
 
+<hr>
+
 ### Pairing
 
-The `pairing` command supports different means regarding Matter device
-commissioning procedure.
+The `pairing` command supports different methods for Matter device commissioning
+procedure. The three most common methods are described in the
+[Using CHIP Tool for Matter device testing](#using-chip-tool-for-matter-device-testing)
+section and are the following:
 
-Thread and Wi-Fi commissioning use cases are described in the
-[Using the CHIP Tool for Matter device testing](#using-chip-tool-for-matter-device-testing)
-section.
+-   `ble-thread` - For Thread commissioning; described under
+    [Commissioning into a Thread network over Bluetooth LE](#commissioning-into-a-thread-network-over-bluetooth-le).
+-   `ble-wifi` - For Wi-Fi commissioning; described under
+    [Commissioning into a Wi-Fi network over Bluetooth LE](#commissioning-into-a-wi-fi-network-over-bluetooth-le)
+-   `onnetwork` - For commissioning the device when it is already present in an
+    IP network; described under
+    [Commissioning with setup PIN code](#commissioning-with-setup-pin-code)
 
-To list all `pairing` sub-commands, run the following command:
+To list all `pairing` sub-commands and commissioning methods, run the following
+command:
 
 ```
 $ ./chip-tool pairing
 ```
 
+#### Attestation-related flags
+
+The `pairing` commissioning command can be run with several flags that allow you
+to modify attestation-related settings:
+
+-   `--paa-trust-store-path` - Use to provide path to the directory that
+    contains the information about Product Attestation Authority (PAA)
+    certificate. The path can be absolute or relative to the current working
+    directory. With this flag, the CHIP Tool looks for the PAA certificate that
+    matches the PAI and the DAC certificates programmed on the device. Without
+    this flag, the CHIP Tool uses the test PAA certificate.
+
+-   `--cd-trust-store-path` - Use to provide path to the directory that contains
+    the information about Certificate Declaration. The path can be absolute or
+    relative to the current working directory. With this flag, the CHIP tool
+    looks for the CD that matches other attestation data on the device. Without
+    this flag, the CHIP Tool uses the test CD.
+
+-   `--only-allow-trusted-cd-keys` - Use to only allow trusted CD verifying keys
```

Comment:
Specifically, this is to be used when you _only_ want the keys from `--cd-trust-store-path` and not the built-in test key.  This dependency between the two flags should be documented.